pages/documentation/developer/repositories/pypowsybl.md

```diff
@@ -0,0 +1,18 @@
+# pypowsybl
+The PyPowSyBl project gives access to PowSyBl framework to Python developers. This Python integration relies on [JPype](https://github.com/jpype-project/jpype). This is not a **native** implementation of PowSyBl features: internally a JVM is started to run the java code, and the Python code access to the Java objects through JNI.
+
+**Reviewers:** [mathbagu](https://github.com/mathbagu), [sylvlecl](https://github.com/sylvlecl)  
+**Committers:** [mathbagu](https://github.com/mathbagu), [sylvlecl](https://github.com/sylvlecl)
+
+## Features
+- Load and Save case file (Supported formats are: [XIIDM](../../grid/formats/xiidm.md))
```

Comment:
Here I understand that "Load" and "Save" are Python terminology ?

Reply:
Using pypowsybl we can access to all PowSyBl features, but I would like to put the focus on the main features a python developer could need. For the moment, the features list match with the shortcuts we have implemented in python ([here](https://github.com/powsybl/pypowsybl/blob/188f5c5bee91300e45bb5e04fcf100da185b2c06/pypowsybl/LoadFlow.py) and [here](https://github.com/powsybl/pypowsybl/blob/188f5c5bee91300e45bb5e04fcf100da185b2c06/pypowsybl/Network.py)).

Do you want I try to add this in the short description?

Reply:
Indeed what I mean is that these name choices "Load" and "Save" do not come from PowSyBl because we prefer ````import```` and ````export`````key word. My remark was just: if you choose ````load````  and ````save```` because it is more natural for Python developper then it is okay for me and a great choice.

Reply:
First, I think there is an inconsistency in the Java API. We have the `Importer` and `Exporter` interfaces that expose an `import` and `export` methods. But we also have the `Importers` and `Exporters` utility classes that are used you hide the complexity introduced by the plugin mechanism. These classes provides a set of `loadNetwork` and `export` methods.

From my point of view, load/save are the word to use here: in my mind import/export suggest you put (or extract) data in your system (i.e. in a database or something) where load (or save) stands for loading a file in memory.

My only requirement is to use the same word for the documentation and for the function's name. Currently, I use : `Network.load(file)` that looks natural to me. Would you prefer to write `Network.import(file)`?